service-bus.tf

```diff
@@ -65,6 +65,15 @@ module "am-orm-judicial-test-pr-subscription" {
   resource_group_name   = local.resource_group_name
 }
 
+module "rd-judicial-subscription-test-pr-subscription" {
+  source                = "git@github.com:hmcts/terraform-module-servicebus-subscription?ref=master"
+  count                 = lower(var.env) == "aat" ? 1 : 0
```

Comment:
You would need this if the sub is only on AAT. If you need on all envs, you can remove this line.

Reply:
Not needed JRD load we are doing trunk based and only aat